appengine_config.py

```diff
@@ -20,9 +20,10 @@
 import os
 import sys
 
-import pkg_resources
 import python_utils
+
 from google.appengine.ext import vendor
+import pkg_resources
```

Comment:
Are these changes needed?

Reply:
Right, lint checks will fail without them because I've fixed the .isort.cfg file.